src/gt4py/next/program_processors/runners/dace_fieldview/gtir_builtin_translators.py

```diff
@@ -379,17 +462,20 @@ def translate_symbol_ref(
     """Generates the dataflow subgraph for a `ir.SymRef` node."""
     assert isinstance(node, gtir.SymRef)
 
-    sym_value = str(node.id)
-    sym_type = sdfg_builder.get_symbol_type(sym_value)
+    sym_name = str(node.id)
+    sym_type = sdfg_builder.get_symbol_type(sym_name)
```

Comment:
The change from value to name is quite significant, can you elaborate why, I am just curious.

Reply:
`node.id` is the symbol name, here meaning a symbol defined in the GT4Py program.